### ISSUE TITLE
Commissioning 60 icons

### ISSUE THREAD

**Issue by tzusman** [tzusman/custom-font-icons#8]
I'm going to be commissioning 60 icons from the icon designer.

My plan is to do about 45 icons from this list: https://github.com/FortAwesome/Font-Awesome/issues?page=1&q=is%3Aopen+label%3A%22new+icon%22+no%3Aassignee+sort%3Acreated-desc+comments%3A%3E9+-label%3A%22brand+icons%22&utf8=%E2%9C%93

And about 15 icons from this list: https://github.com/FortAwesome/Font-Awesome/issues?q=is%3Aopen+label%3A%22brand+icons%22+no%3Aassignee+comments%3A%3E24+sort%3Acomments-desc

@CrashyBang would love your input on this.  the 45/15 idea is just a first stab, as well as the source of the icon ideas.

I figure that with 100 icons in the set, hopefully we'll garner more interest and community involvement.

**Comment by CrashyBang** [comment]
@tzusman I think that's a good idea, for the 15 brand icons I would steer clear of 'programming' related icons as most of those can be found in packages like this: http://vorillaz.github.io/devicons/#/main

As for a build tool? are you imaging some sort of npm/gulp build tool that will iterate over the font files and generate the CSS? sorry I am just a little unclear on the expected functionality.

Cheers,
Otis.

**Comment by tzusman** [comment]
@CrashyBang Good call on not including programming icons.

Checkout [feature/build](https://github.com/tzusman/custom-font-icons/tree/feature/build)'s gulpfile.coffee for how I'm addressing the build process

**Comment by CrashyBang** [comment]
Hey @tzusman looks good well let me know if I can help with anything :) gulpfile looks nice :+1: 

Cheers.

**Comment by tzusman** [comment]
@CrashyBang do you mind making similar changes to the feature/build branch so that the font awesome structure is preserved in the build directory?

**Comment by CrashyBang** [comment]
@tzusman Yeah of course sounds good to me :). So is there anything I can do now?

Cheers.

**Comment by tzusman** [comment]
@CrashyBang  I mean - besides for the font awesome structure, anything you think that would improve the project, I'd love the contribution!  Need to start putting together an github.io page that can be automatically generated and deployed through the gulp pipeline to show all the icons and docs.

**Comment by CrashyBang** [comment]
Hey @tzusman sweet I will look at starting that when I can I also have another PR I am working on for another project so it may take a little while :L

Cheers.

**Comment by tzusman** [comment]
@CrashyBang no worries! thanks for your help
